README.md

```diff
@@ -1,2 +1,88 @@
-# yotas
-Oss's monetisation system
+# Yotas
+
+Oss's reward system
+
+[French](#french)
+
+[English](#english)
+
+# english
+
+Yotas is OSS's reward system, we would like our open source contributors across the organisation to be rewarded for the effort and energy they put into the platform. Each interaction they have on GitHub will give them a certain number of yotas which they can exchange for goodies in an OSS shop, mentoring, ebooks, etc....
+
+## Why?
+Contributing to open source takes some effort, and we would like every contributor to be rewarded for the time and energy they put into helping our community.
```

Comment:
We should change all rewards to awards - the Admins determine (award) what is given to the contributor for the job done

Reply:
done